llvm/test/CodeGen/RISCV/GlobalISel/irtranslator/vec-args.ll

```diff
@@ -0,0 +1,1233 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+; RUN: llc -mtriple=riscv32 -mattr=+v -global-isel -stop-after=irtranslator \
```

Comment:
To test bfloat16 and half correctly you need to add -mattr=+experimental-zvfbfmin,+zvfh